src/openforms/scss/admin/_admin_theme.scss

```diff
@@ -166,6 +166,10 @@ div.breadcrumbs {
   }
 }
 
+#changelist table thead th {
+  padding: 8px;
+}
```

Comment:
this should not be a global override, but instead scoped to your own table classes where the problem is happening. Other list pages don't have this padding problem.

I'm also wondering if it doesn't make more sense to mimic the Django admin list markup for table headers instead of overriding the CSS. I believe they put some `div`s inside of the `th` normally.

Reply:
Yes it could be a better approach